apps/bare-expo/MainNavigator.tsx

```diff
@@ -53,40 +43,76 @@ const Redirect = optionalRequire(() =>
 ) as any;
 
 if (NativeComponentList) {
-  routes.ExpoApis = NativeComponentList.ExpoApis;
-  routes.ExpoComponents = NativeComponentList.ExpoComponents;
+  routes.apis = NativeComponentList.apis;
+  routes.components = NativeComponentList.components;
 }
 
-const MainNavigator = createBottomTabNavigator(routes, {
-  initialRouteName: 'TestSuite',
-  tabBarOptions: {
-    labelStyle: {
-      fontSize: 12,
-    },
-    activeTintColor: Colors.activeTintColor,
-    inactiveTintColor: Colors.inactiveTintColor,
-    safeAreaInset: {
-      top: 5,
-      right: 'always',
-      bottom: 'always',
-      left: 'always',
+const Tab = createBottomTabNavigator();
+
+const Switch = createStackNavigator();
+
+const linking = {
+  prefixes: [Platform.select({ web: Linking.makeUrl('/'), default: 'bareexpo://' })],
+  config: {
+    main: {
+      path: '',
```

Comment:
There was slight change in this config so would be good to upgrade to latest versions of navigation to use those  :)

then it'll be like:

```js
config: {
  screens: {
    main: ...
  }
}
```